### PR TITLE
#7 - Create DTO class

### PR DESCRIPTION
**Pull request by Andreymlc**
Created the StudentDto and StudentMappers class

### CODE REVIEW

File: api/Dto/StudentDto.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace api.Dto;
+
+public class StudentDto{
+    public required string StudentId { get; set; }
+    public required string Email { get; set; }
+    public required string FullName { get; set; }
+    public required string Password { get; set; }
+    public required string Group { get; set; }
+}
```
Comment (NN171) [inline api/Dto/StudentDto.cs:9]: In the end of the file always adding the new line

File: api/Mappers/StudentMappers.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            return new StudentDto{
+                StudentId = studentmodel.StudentId,
+                Email = studentmodel.Email,
+                FullName = studentmodel.FullName,
+                Password = studentmodel.Password,
+                Group = studentmodel.Group
+            };
+        }
+    }
+}
```
Comment (NN171) [inline api/Mappers/StudentMappers.cs:16]: The same comment

File: api/Dto/StudentDto.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+namespace api.Dto;
+
+public class StudentDto{
+    public required string StudentId { get; set; }
```
Comment (NN171) [inline api/Dto/StudentDto.cs:4]: My bad, I have forgotten, that we are using StudentId as int variable

**Review by NN171 — COMMENTED** [review]
Change it, and u can merge

File: api/Dto/StudentDto.cs (hunk 1 above)
Comment (NN171) [reply api/Dto/StudentDto.cs:9]: You haven't still added the new line

File: api/Mappers/StudentMappers.cs (hunk 2 above)
Comment (NN171) [inline api/Mappers/StudentMappers.cs:16]: The same thing here

**Review by NN171 — APPROVED** [review]

File: api/Mappers/StudentMappers.cs (hunk 2 above)
Comment (Andreymlc) [reply api/Mappers/StudentMappers.cs:16]: Code

Comment (Andreymlc) [reply api/Mappers/StudentMappers.cs:16]: This is strange

Comment (NN171) [reply api/Mappers/StudentMappers.cs:16]: Ok

**Pull request merged**